### PR TITLE
Carbon fiber sabres

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds in the carbon fiber saber and sheath, available from cargo for 800 credits.
The carbon fiber saber is a weapon designed for security containing absolutely no electronic components.
While not as effective at penetrating armour than the officer's saber (only 25 compared to 75), it has a higher base damage of 17 instead of 15. Additionally, it does not passively block and requires walking for the blocking to happen.
Also it looks cool.

The saber was originally used by Nanotrasen space forces when projectile based weapons were ineffective in space, however with the advancement of technology and laser based weaponry, they are out of regular use now.

1 spawns in the HOS's locker.

image

## Why It's Good For The Game

A tool that allows security to counter EMP based threats, ordered from cargo.

## Testing Photographs and Procedure

image

image

image

## Changelog
:cl:
add: Adds in the carbon fiber saber, a security saber purchasable from cargo.
/:cl:

### CODE REVIEW

**Comment by VictorVow** [comment]
Why would someone order this over a combat knife that can fit in your backpack or boot? Why would you order either when in the proposed case of lings/cultists you'd just order ballistics? This would never be ordered, I can only see this maybe getting use if it spawned in the armory.

**Comment by PowerfulBacon** [comment]
> Why would someone order this over a combat knife that can fit in your backpack or boot? Why would you order either when in the proposed case of lings/cultists you'd just order ballistics? This would never be ordered, I can only see this maybe getting use if it spawned in the armory.

Looks cooler, better against armoured opponents.

**Comment by VictorVow** [comment]
25% armour penetration is nothing tbh, especially for a bulky item. And if you're dealing with lings/cultists again, you'd want to fight at range with ballistics. Looks cool, but again as a cargo crate it sucks compared to combat knives even.

**Comment by ToasterBan** [comment]
> Why would someone order this over a combat knife that can fit in your backpack or boot? Why would you order either when in the proposed case of lings/cultists you'd just order ballistics? This would never be ordered, I can only see this maybe getting use if it spawned in the armory. 

if this gets merged I will buy this every round out of sheer drip factor

File: code/game/objects/items/melee/misc.dm [hunk 1]
```diff
@@ -145,15 +144,20 @@
...
+	item_state = "sabre_fiber"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	obj_flags = NONE
+	force = 17	//1 damage weaker than the nullrod and energy pen
+	//Walking is required for blocking
+	block_level = 0
+	block_upgrade_walk = 2
+	block_power = 30
+	armour_penetration = 25
```
Comment (PowerfulBacon) [inline code/game/objects/items/melee/misc.dm]:
```suggestion
	armour_penetration = 45
```

**Comment by PowerfulBacon** [comment]
Armour penetration increased to 45, making it more effective than the knife against targets with armour >=30, but still weaker than the captain's saber against targets with armour >= 40

**Comment by PowerfulBacon** [comment]
image

**Comment by RDS88-beeman** [comment]
GIVE ME THE OBTAINABLE SWORD
PLEASE
like what swords do we even have
we have one on cap, one on curator, a bunch of variants on chap of which he can get one, and a bunch stuck in the thunderdome
and that's it for sword aesthetics
give me the red blade

**Comment by FriendlyContractor** [comment]
Would it be too much to ask to just have just both sabers buffed to deal 20 damage with 75 armor penetration ? Its not like its gonna become a new meta over a shotgun but it least would be somewhat viable. Its super cool but i completly cannot imagine anybody using this bulky item for anything else than its coolnes factor in its current state

**Comment by FriendlyContractor** [comment]
This would also give this weapon a unique niche of being armor buster uselfull in rare case somebody manages to stack a lot of armor.

**Comment by PowerfulBacon** [comment]
Maybe the blocking should be buffed to 1,0

**Comment by VictorVow** [comment]
Yeah higher armour penetration or increased blocking seems like a good niche for sabers compared to combat knives.

**Comment by KennyBadrige** [comment]
some wack ass changes are being made, i dont even mind. Go nuts

**Comment by Isy232** [comment]
it looks cool and swords are a meme anyways.

swords aren't the best option in almost all situations and are a backup / gimmick roleplay weapon.

giving security cool ass swords sounds fun, also props for making them cargo orderable.

knives are for edgy "rogues"

swords are for men.

**Comment by PowerfulBacon** [comment]
> swords are a meme anyways.

Depends how robust you are 😎

File: code/game/objects/items/melee/misc.dm [hunk 2]
```diff
@@ -145,15 +144,20 @@
...
+	desc = "A sabre made of a reinforced carbon-fiber polymer. While lighter and sharper than its monomolecular counterpart, it has a tougher time penetrating armour."
+	icon_state = "sabre_fiber"
+	item_state = "sabre_fiber"
+	lefthand_file = 'icons/mob/inhands/weapons/swords_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/swords_righthand.dmi'
+	obj_flags = NONE
+	force = 17	//1 damage weaker than the nullrod and energy pen
+	//Walking is required for blocking
+	block_level = 0
+	block_upgrade_walk = 2
```
Comment (PowerfulBacon) [inline code/game/objects/items/melee/misc.dm:158]:
```suggestion
	block_level = 1
	block_upgrade_walk = 0
```

File: code/game/objects/items/storage/belt.dm [hunk 3]
```diff
@@ -783,6 +783,16 @@
 	new /obj/item/melee/sabre(src)
 	update_icon()
 
+/obj/item/storage/belt/sabre/fiber
+	name = "carbon-fiber saber sheath"
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	name = "carbon-fiber sabre sheath"
```

File: code/game/objects/items/storage/belt.dm [hunk 4]
```diff
@@ -783,6 +783,16 @@
...
+	desc = "A military grade saber sheath. While once used widely by Nanotrasen space forces due to its reliability in space, the advancement in energy weapons have made them redundant."
```
Comment (PowerfulBacon) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
	desc = "A military grade sabre sheath. While once used widely by Nanotrasen space forces due to its reliability in space, the advancement in energy weapons have made them redundant."
```

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -344,6 +344,14 @@
 					/obj/item/gun/energy/disabler)
 	crate_name = "disabler crate"
 
+/datum/supply_pack/security/saber
+	name = "Carbon Fiber Saber"
```
Comment (PowerfulBacon) [inline code/modules/cargo/packs.dm]:
```suggestion
	name = "Carbon Fiber Sabre"
```

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -344,6 +344,14 @@
...
+	desc = "A single carbon-fiber saber and sheath. Highly reliable, sharp and contains no electronic components."
```
Comment (PowerfulBacon) [inline code/modules/cargo/packs.dm]:
```suggestion
	desc = "A single carbon-fiber sabre and sheath. Highly reliable, sharp and contains no electronic components."
```

**Comment by heepox** [comment]
so just a drip item with no real utility for anything but melee fights with carp users who have anti stun?
worse than just bola baton which is the superior form of sec-fu.

**Comment by ToasterBan** [comment]
> so just a drip item with no real utility for anything but melee fights with carp users who have anti stun?
> worse than just bola baton which is the superior form of sec-fu.

its emp proof unlike bolas or batong, also has blocking and lethal damage

**Comment by Archanial** [comment]
:key:

**Pull request closed without merging**